docker-compose.yml

```diff
@@ -9,14 +9,17 @@ services:
       interval: 10s
       retries: 10
     restart: always
+    ports:
+      - 5432:5432
     environment:
-      - POSTGRES_USER_FILE=/run/secrets/focalboard-db
-      - POSTGRES_PASSWORD_FILE=/run/secrets/focalboard-db
-      - POSTGRES_DB_FILE=/run/secrets/focalboard-db
+      - POSTGRES_USER_FILE=/run/secrets/db_user
+      - POSTGRES_PASSWORD_FILE=/run/secrets/db_passwd
+      - POSTGRES_DB=boards
```

Comment:
why not use secret for postgres db name as well?

Reply:
Hi @st-dav,

The README mentions a "production ready" environment. It should contain the specifics of this environment, that is: you already have a reverse proxy. As a user, when I tried the docker-compose, it didn't work because of DB issues and no web proxy. The README could provide a topology of the environment so other users can deploy a web proxy. Maybe you can have 2 branches like "Complete env" and "No proxy included"? Just to keep your users in the right track. :-)

I removed the DB name from the secrets because it's not really added security IMO. What I mean is that if an attacker gets into a position he can access the DB, it's a one SQL query to list all the available DBs, then all the tables. If the app is vulnerable to SQL injection, then it can be enumerated as well, so that's not a secret per se. But I understand it can be kept as a secret, why not? I'd split it from the one file thgough, keep each one separate instead of a one file with all info.

Alex.